src/Microsoft.DotNet.XHarness.iOS.Shared/TestReporter.cs

```diff
@@ -1,4 +1,5 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+﻿#nullable enable
```

Comment:
Doing the Lord's work 😊

Reply:
However, I am wondering if it should go above the license

Reply:
I don'g know to be honest.